### PR TITLE
Updated Gatekeeper Guard

### PR DESCRIPTION
**Pull request by araptarchis**
The docs where only mentioning the Captcha challenge, which is only one of the challenges that are actually supported. I added a table of the other, more advanced options.

### CODE REVIEW

**Comment by staccDOTsol** [comment]
It sounds like you're referring to a documentation update for a Gatekeeper Guard. What is your question or concern about this update?

**Comment by araptarchis** [comment]
> It sounds like you're referring to a documentation update for a Gatekeeper Guard. What is your question or concern about this update?

No question or concern from my side, I just updated the documentation.

**Comment by MarkSackerberg** [comment]
Thank you for your update @araptarchis !
As far as i know the gatekeeper technically does not has to be civic, so at least having a sentence in there mentioning it would make sense from my point of view to keep it a bit more general.

**Comment by samuelvanderwaal** [comment]
Is this read to be merged or does it still need the extra sentence added? @MarkSackerberg @araptarchis

File: docs/01-programs/02-candy-machine/09-available-guards/07-gatekeeper.md [hunk 1]
```diff
@@ -4,21 +4,24 @@ description: "Restricts minting via a Gatekeeper Network e.g. Captcha integratio
 
 import { Accordion, AccordionItem } from '/src/accordion.jsx';
 
-# Gatekeeper
+# Gatekeeper / Civic Pass
```
Comment (lorisleiva) [inline docs/01-programs/02-candy-machine/09-available-guards/07-gatekeeper.md:7]: Please revert this as the guard is called `Gatekeeper`.

```suggestion
# Gatekeeper
```

File: docs/01-programs/02-candy-machine/09-available-guards/07-gatekeeper.md [hunk 2]
```diff
@@ -4,21 +4,24 @@ description: "Restricts minting via a Gatekeeper Network e.g. Captcha integratio
...
+| **Identity Document Verification** ([Demo](https://verify.civic.com)) 	| ID Document Authenticity  - Selfie matches ID Document photo - 18+ age check - Email - Liveness check - Location - Banned countries check: OFAC 	| [Reach out to Civic](https://share.hsforms.com/1Z4QgWNh0RN2-81jJDcrN2Qbzn0a) 	|
+| **Custom Verification**                                             	    | Additional specific countries, on-chain checks or other verifications.                                                                          	| [Reach out to Civic](https://share.hsforms.com/1NvBk0zfyR3aWcMosBxJETQbzn0a) 	|
 
-Here are some additional recommended materials you may find helpful when setting up a Gatekeep Network.
+ The Captcha Verification is free to use without any restrictions, which is what we will be using in this guide. Other verification types involve Personal Identifiable Information (PII) are require you to reach out to Civic first.
 
-- [The CIVIC Documentation](https://docs.civic.com/civic-pass/overview)
-- [Gateway JS Library](https://www.npmjs.com/package/@identity.com/solana-gateway-ts)
-- [Gateway React Components](https://www.npmjs.com/package/@civic/solana-gateway-react)
```
Comment (lorisleiva) [inline docs/01-programs/02-candy-machine/09-available-guards/07-gatekeeper.md:21]: Any reason these links were removed?

File: docs/01-programs/02-candy-machine/09-available-guards/07-gatekeeper.md [hunk 3]
```diff
@@ -4,21 +4,24 @@ description: "Restricts minting via a Gatekeeper Network e.g. Captcha integratio
...
 
 ## Overview
 
-The **Gatekeeper** guard checks whether the minting wallet has a valid **Gateway Token** — also known as a **Civic Pass** — from a specified **Gatekeeper Network**.
+The **Gatekeeper** guard checks whether the minting wallet has a valid **Gateway Token** — also known as a **Civic Pass**. There are various types of Civic Pass you can use to guard your mint, ranging from completing a simple Captcha challenge, all the way to capturing a 3D video selfie that guarantees a "one NFT per human" mint. 
 
-In most cases, this token will be obtained after completing a Captcha challenge but any Gatekeeper Network may be used.
+ You select the type of Civic Pass to use by configuring the corresponding [**Gatekeeper Network**](https://docs.civic.com/civic-pass/overview/how-it-works#the-on-chain-gatekeeper-network) in the Gatekeeper guard.
 
-There isn’t much to set up on the Candy Machine side but, depending on the selected Gatekeeper Network, you may need to ask the minting wallet to perform so pre-validation checks to grant them the required Gateway Token.
```
Comment (lorisleiva) [inline docs/01-programs/02-candy-machine/09-available-guards/07-gatekeeper.md:15]: Not a big fan of this change as the transition with the previous paragraph is a bit harsh. May I suggest the following instead?


> In order to add a **Gatekeeper** to your Candy Machine, you must first select a **Gatekeeper Network** which defines how user verification is performed.
> 
> There isn’t much to set up on the Candy Machine side, you simply need to provide the public key of the Gatekeeper Network of your choice and whether or not Gateway Tokens should expire after being used. Note that depending on the selected Gatekeeper Network, you may need to ask the user to perform so pre-verification checks to grant them the required Gateway Token.
> 
> Here is a list of available Gatekeeper Networks and how to get started with them.

File: docs/01-programs/02-candy-machine/09-available-guards/07-gatekeeper.md [hunk 4]
```diff
@@ -4,21 +4,24 @@ description: "Restricts minting via a Gatekeeper Network e.g. Captcha integratio
...
+| **Civic Pass**                                                      	    | **Features**                                                                                                                                    	| **Gatekeeper Network**                                                       	|
+|---------------------------------------------------------------------	    |-------------------------------------------------------------------------------------------------------------------------------------------------	|------------------------------------------------------------------------------	|
+| **Captcha Verification** ([Demo](https://getpass.civic.com))         	| Captcha check to filter bots.                                                                                                                   	| ignREusXmGrscGNUesoU9mxfds9AiYTezUKex2PsZV6                                  	|
```
Comment (lorisleiva) [inline docs/01-programs/02-candy-machine/09-available-guards/07-gatekeeper.md:17]:
```suggestion
| **Captcha Verification** ([Demo](https://getpass.civic.com))         	| Captcha check to filter bots.                                                                                                                   	| `ignREusXmGrscGNUesoU9mxfds9AiYTezUKex2PsZV6`                                  	|
```

**Review by lorisleiva — CHANGES_REQUESTED** [review]

**Comment by lorisleiva** [comment]
Hi there, I'm closing this due to inactivity and the fact that we moved our docs to developers.metaplex.com (https://github.com/metaplex-foundation/developer-hub).

**Pull request closed without merging**